src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -26,251 +29,145 @@ the MIT License. See LICENSE in the project root for license information. -->
+                        <TextBox x:Uid="Profile_Icon"
+                                 Text="{x:Bind State.Profile.Icon, Mode=TwoWay}"/>
+                        <Button x:Uid="Profile_IconBrowse"
+                                Margin="10,0,0,0"/>
+                    </StackPanel>
+                    <TextBox x:Uid="Profile_TabTitle"
+                             Text="{x:Bind State.Profile.TabTitle, Mode=TwoWay}"
+                             HorizontalAlignment="Left"/>
+                    <Controls:RadioButtons x:Uid="Profile_ScrollbarVisibility"
+                                           ItemsSource="{x:Bind ScrollStateList, Mode=OneWay}"
+                                           SelectedItem="{x:Bind CurrentScrollState, Mode=TwoWay}"
+                                           ItemTemplate="{StaticResource EnumRadioButtonTemplate}"/>
+                </StackPanel>
+            </ScrollViewer>
+        </PivotItem>
 
+        <!-- Appearance Tab -->
+        <PivotItem x:Uid="Profile_Appearance">
+            <ScrollViewer>
+                <StackPanel HorizontalAlignment="Left">
+                    <TextBox x:Uid="Profile_FontFace"
+                             Text="{x:Bind State.Profile.FontFace, Mode=TwoWay}"/>
+                    <Controls:NumberBox x:Uid="Profile_FontSize"
+                                        Value="{x:Bind State.Profile.FontSize, Mode=TwoWay}"
+                                        SmallChange="1"
+                                        LargeChange="10"/>
+                    <TextBox x:Uid="Profile_FontWeight"/>
+                    <TextBox x:Uid="Profile_Padding"
+                             Text="{x:Bind State.Profile.Padding, Mode=TwoWay}"/>
+                    <Controls:RadioButtons x:Uid="Profile_CursorShape"
+                                           ItemsSource="{x:Bind CursorShapeList, Mode=OneWay}"
+                                           SelectedItem="{x:Bind CurrentCursorShape, Mode=TwoWay}"
+                                           ItemTemplate="{StaticResource EnumRadioButtonTemplate}"/>
+                    <Controls:NumberBox x:Uid="Profile_CursorHeight"
+                                        Value="100"
+                                        SmallChange="1"
+                                        LargeChange="10"/>
+                    <ComboBox x:Uid="Profile_ColorScheme">
+                        <ComboBox.Items>
+                            <ComboBoxItem Content="Campbell"/>
+                            <ComboBoxItem Content="Campbell Powershell"/>
+                            <ComboBoxItem Content="Vintage"/>
+                            <ComboBoxItem Content="One Half Dark"/>
+                            <ComboBoxItem Content="One Half Light"/>
+                            <ComboBoxItem Content="Tango Dark"/>
+                            <ComboBoxItem Content="Tango Light"/>
+                        </ComboBox.Items>
+                    </ComboBox>

```

Comment:
these controls get _labels_, right? You know it's a color scheme, not just "lol campbell is in this box"?

Reply:
This isn't hooked up yet either. #8137 covers actually binding this but I figured I'd go ahead and make this wholistically look better independently of adding the correct functionality. _Maybe_ I could do this in the same PR as BIAlignment binding/converter?

Reply:
I'm asking you if this control has a label.

Reply:
Oh! You mean a header like this:
![image](https://user-images.githubusercontent.com/11050425/99298783-148f2000-27ff-11eb-835f-864b503b3831.png)
Yeah, that's all handled in the resw.